examples/kubernetes/generator/generate_secrets.sh

```diff
@@ -1,55 +1,73 @@
 #!/bin/bash
 
+TEMPLATE_ENGINE=jinja
+
+proc-template() {
```

Comment:
I have done so and squashed the changes in the original commit.

Didn't see this as I use a tab size of 2, sorry.